### PR TITLE
feat(weave): add number of rows for ST in home page table

### PR DESCRIPTION
**Pull request by jlzhao27**
Add a new column for number of rows for stream tables:
image

Note: this does not work for logged tables so I display a `-` to indicate the information does not exist

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx [hunk 1]
```diff
@@ -537,6 +538,11 @@ const CenterProjectTablesBrowser: React.FC<
       'updated at': moment.utc(b.updatedAt).local().calendar(),
       'created at': moment.utc(b.createdAt).local().calendar(),
       'created by': b.createdByUserName,
+      // we use the # of steps in the run history to compute rows quickly. This works for stream tables
+      // because the table leverages run history. For logged tables, we must open to table in order to
+      // to know the number of rows which is a much more expensive operation. Instead, we will just
+      // display a dummy placeholder
+      'number of rows': '-',
```
Comment (jlzhao27) [inline weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx:545]: :(

**Review by tssweeney — APPROVED** [review]

File: weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx [hunk 2]
```diff
@@ -537,6 +538,11 @@ const CenterProjectTablesBrowser: React.FC<
       'updated at': moment.utc(b.updatedAt).local().calendar(),
       'created at': moment.utc(b.createdAt).local().calendar(),
       'created by': b.createdByUserName,
+      // we use the # of steps in the run history to compute rows quickly. This works for stream tables
+      // because the table leverages run history. For logged tables, we must open to table in order to
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx:542]: open to table => open the table

File: weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx [hunk 3]
```diff
@@ -537,6 +538,11 @@ const CenterProjectTablesBrowser: React.FC<
...
+      // to know the number of rows which is a much more expensive operation. Instead, we will just
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx:543]: you have "to" at the end of the previous line

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx (hunk 2 above)
Comment (jlzhao27) [reply weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx]: fixed

File: weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx (hunk 3 above)
Comment (jlzhao27) [reply weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx:543]: womp, i always self review my code and never my comments :P fixed

**Pull request merged**